content/en/docs/reference/glossary/node.md

```diff
@@ -14,5 +14,5 @@ tags:
 
 <!--more--> 
 
-A worker machine may be a VM or physical machine, depending on the cluster. It has the {{< glossary_tooltip text="Services" term_id="service" >}} necessary to run {{< glossary_tooltip text="Pods" term_id="pod" >}} and is managed by the master components. The {{< glossary_tooltip text="Services" term_id="service" >}} on a node include Docker, kubelet and kube-proxy.
+A worker machine may be a VM or physical machine, depending on the cluster. It has the Kubernetes services necessary to run {{< glossary_tooltip text="Pods" term_id="pod" >}} and is managed by the master components. The services on a node include {{< glossary_tooltip text="container runtime interface" term_id="cri" >}}, {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} and {{< glossary_tooltip text="kube-proxy" term_id="kube-proxy" >}}.
```

Comment:
Nodes need to have a container runtime. CRI is the interface that any runtime must promise to implement in order to run Pods / containers.